browser/js/signup/signup.html

```diff
@@ -0,0 +1,49 @@
+<form id="login-form" name="signupForm" ng-submit="signupForm.$valid && createUser(userInfo)">
+    <h1>Signup</h1>
+    
+    <alert type="danger" ng-show="error">
+        {{ error }}
+    </alert>
+    <div class="form-group">
+        <label for="signup-first_name">First Name</label>
+        <input type="text" ng-model="userInfo.first_name" class="form-control" id="signup-first_name" placeholder="First Name">
+    </div>
+    <div class="form-group">
+        <label for="signup-last_name">Last Name</label>
+        <input type="text" ng-model="userInfo.last_name" class="form-control" id="signup-last_name" placeholder="Last Name">
+    </div>
+    <div class="form-group">
+        <label for="email">Email address</label>
+        <input type="text" ng-model="userInfo.email" class="form-control" id="email" placeholder="Enter email">
+    </div>
+    <div class="form-group">
```

Comment:
Can we move email address to the top and make it required, since it's the only required field?